iron-form.html

```diff
@@ -130,6 +173,15 @@
         valid = el.validate() && valid;
       }
       return valid;
+    },
+
+    _useValue: function(el) {
+      // Checkboxes and radio buttons should only use their value if they're checked.
+      if (el.type !== 'checkbox' && el.type !== 'radio') {
```

Comment:
The problem is that the native checkbox can't override this. The only elements that currently pose this problem, really, are the native ones. (the `paper-checkbox` can just set a civilized true or false in `value`, for example)